lib/presenters/pull_requests_by_application.rb

```diff
@@ -5,13 +5,14 @@ def execute(ungrouped_pull_requests)
         value.fetch(:application_name)
       end
 
-      grouped_pull_requests.map do |application_name, pull_requests|
+      pull_requests_by_application = grouped_pull_requests.map do |application_name, pull_requests|
```

Comment:
Very minor, but not a fan of this variable name.  I'd probably get around needing to name this variable by extracting methods.  I'm sure you could come up with better names.

```
sort_by_app_name(sort_by_pr_title(grouped_pull_requests))
```